pubsublite/streaming-analytics/src/main/java/examples/PubsubliteToGcs.java

```diff
@@ -114,9 +114,8 @@ public static void main(String[] args) throws InterruptedException {
                 .accumulatingFiredPanes())
         .apply("Write elements to GCS", new WriteOneFilePerWindow(options.getOutput(), numShards));
 
-    // Execute the pipeline with an optional timeout for your local program.
-    // This will not cancel the remote job.
-    pipeline.run().waitUntilFinish(Duration.standardMinutes(10));
+    // Execute the pipeline.
+    pipeline.run();
```

Comment:
It would be nice to have a comment mentioning that a Flex Template expects to only run, and not waitUntilFinish.

Reply:
Done.